mappings/net/minecraft/nbt/visitor/NbtOrderedStringFormatter.mapping

```diff
@@ -0,0 +1,30 @@
+CLASS net/minecraft/class_5625 net/minecraft/nbt/visitor/NbtOrderedStringFormatter
```

Comment:
Is this used specifically in pretty-printing like with the functions in `NbtHelper`? Would it make sense for the names to be similar?

Reply:
It's almost exactly like `NbtTextFormatter` except that it sorts the keys in a janky way.